OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/AbstractHookActivity.java

```diff
@@ -28,6 +28,13 @@
  * as we need to write into the SD card and automation will parse this file.
  */
 public abstract class AbstractHookActivity extends Activity implements Observer, DefaultHardwareBackBtnHandler {
+
+	public static final String EXTRA_EMBED_CODE = "embed_code";
+	public static final String EXTRA_PCODE = "pcode";
+	public static final String EXTRA_DOMAIN = "domain";
+	public static final String EXTRA_AUTO_PLAY = "autoPlay";
+	public static final String EXTRA_SELECTED_FORMAT = "selectedFormat";
```

Comment:
Can be package-private

Reply:
Or even better to move `IntentBuilder` somewhere to helper package or something

Reply:
moved to utils package